scripts/pathseq/wdl/pathseq_pipeline.wdl

```diff
@@ -20,179 +26,712 @@
 ## - - one or more read groups all belong to a single sample (SM)
 ##
 ## Output:
-## - BAM file containing microbe-mapped reads and reads of unknown sequence
-## - Tab-separated value (.tsv) file of taxonomic abundance scores
-## - Picard-style metrics files for the filter and scoring phases of the pipeline
+## - non_host_paired_bam, non_host_unpaired_bam: BAM files containing quality-filtered non-host paired and unpaired reads
```

Comment:
Does this imply that there are samples with a mix of paired and unpaired reads, e.g. 2x150 and 1x300?  Or does unpaired mean "not properly paired" as in the SAM flags?

Reply:
I think PathSeq assumes the input is either all paired or unpaired, but many single mates get lost during filtering.